defaults/main.yml

```diff
@@ -1,16 +1,22 @@
 ---
 coredns_version: 1.8.3
 coredns_download_link: "https://github.com/coredns/coredns/releases/download/v{{ coredns_version }}/coredns_{{ coredns_version }}_linux_{{ go_arch }}.tgz"
-coredns_dns_port: 53
-
-# Config file name, searched in ansible templates path.
-coredns_config_file: ""
 
 # Use a local binary instead of the official release.
 coredns_remote_download: True
 coredns_binary_local_dir: "/tmp"
-
+coredns_tarball_local_path: "/tmp/coredns_{{ coredns_version }}_linux_{{ go_arch }}.tgz"
```

Comment:
It's a bit odd to keep the whole file name the same and change only extension from `tar.gz` to `tgz`. This will cause issues in case of significant download link change (`tar.bz2` for example), you'll need to update `coredns_tarball_local_path` too.

Not sure if putting filename in third variable and use it in both `coredns_download_link` and `coredns_tarball_local_path` is a solution.

I ain't mind too much to leave it as is.

Reply:
This path was copy-pasted from original role, I'd like to leave it as is.

Reply:
OK